runtime/testing/browser/test-utility.ts

```diff
@@ -0,0 +1,23 @@
+export interface SkyAppTestUtilityEventArgs {
+  bubbles?: boolean;
+  cancelable?: boolean;
+}
+
+function getWindow() {
```

Comment:
I'm not sure I understand the purpose of this function.  I could see if maybe you wanted to mock out the `window` object you'd want to do something like this, but it's not possible with the code as it is.